src/dzitilesource.js

```diff
@@ -197,10 +197,10 @@ $.extend( $.DziTileSource.prototype, $.TileSource.prototype, /** @lends OpenSead
             xMax = xMin + rect.width * scale;
             yMax = yMin + rect.height * scale;
 
-            xMin = Math.floor( xMin / this.tileSize );
-            yMin = Math.floor( yMin / this.tileSize );
-            xMax = Math.ceil( xMax / this.tileSize );
-            yMax = Math.ceil( yMax / this.tileSize );
+            xMin = Math.floor( xMin / this._tileWidth );
+            yMin = Math.floor( yMin / this._tileWidth );
```

Comment:
Wow, good catch!

Even though dzi tiles are square by definition, might as well make the yMin and yMax based on _tileHeight instead of _tileWidth.

Otherwise looks great!